osu.Game/Overlays/Dashboard/Home/News/NewsTitleLink.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Platform;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Online.API.Requests.Responses;
+
+namespace osu.Game.Overlays.Dashboard.Home.News
+{
+    public class NewsTitleLink : OsuHoverContainer
+    {
+        private readonly APINewsPost post;
+
+        public NewsTitleLink(APINewsPost post)
+        {
+            this.post = post;
+
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(GameHost host)
+        {
+            Child = new TextFlowContainer(t =>
```

Comment:
I know we don't do underline text anywhere but the current yellow colour on hover definitely feels wrong. Would use something themed instead if possible.

Reply:
Would be nice to have any suggestions

Reply:
Alright, I just changed hover colour
![Screenshot_1](https://user-images.githubusercontent.com/22874522/89927954-ca9b0a80-dc0f-11ea-971c-2bfce1254384.png)